### ISSUE TITLE
Qwen 2.5 VL : custom loss function , not able to find where the loss is implemented

### ISSUE THREAD

**Issue by akhila1729** [modelscope/ms-swift#6026]
The bash script I am using is : 
```

NPROC_PER_NODE=8 \
CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7,8 \
swift sft \
    --model $MODEL_PATH_IL_SECOND_STAGE \
    --run_name 'Qwen2.5-VL-7B-Instruct_nodule_data_llm_train_IL_CL_stage_3_v5_mask_score_sept_23' \
    --train_type full \
    --max_pixels $((1568 * 1568)) \
    --dataset $DATASET_PATH_IL_THIRD_STAGE \
    --dataset_shuffle True \
    --new_special_tokens /home/users/akhila.krishna/ms-swift/examples/train/liger/token_list.txt\
    --packing false \
    --max_new_tokens 16000 \
    --split_dataset_ratio 0.05 \
    --torch_dtype bfloat16 \
    --per_device_train_batch_size 1 \
    --per_device_eval_batch_size 1 \
    --learning_rate 1e-5 \
    --freeze_vit true \
    --freeze_aligner true \
    --num_train_epochs 1 \
    --gradient_accumulation_steps 4 \
    --eval_steps 9000 \
    --save_steps 500 \
    --logging_steps 5 \
    --max_length 32000 \
    --warmup_ratio 0.05 \
    --dataloader_num_workers 16 \
    --dataset_num_proc 8 \
    --save_total_limit 20 \
    --save_only_model false \
    --output_dir /raid/akhila/mswift/sft/ckpts/Qwen2.5-VL-7B-Instruct_nodule_data_llm_train_IL_CL_stage_3_v5_mask_score_sept_23_debug \
    --deepspeed zero2 \
    --attn_impl flash_attn \
    --use-hf true \
    --system 'You are a helpful medical assistant.' \
    --use_liger_kernel true \
    --report_to wandb \
    --create_checkpoint_symlink true

```

I wanted to add one more function to the current cross entropy loss function of the auto regressive model. I cannot find where the code is implemented. I tried using pdb.set_trace(), but couldn't find which code is running. 
/raid/akhila/conda_envs/mswift/lib/python3.10/site-packages/transformers/models/qwen2_5_vl/modeling_qwen2_5_vl.py

I put pdb.set_trace() in this file and checked, but this file is not called at all. 

My questions : 
1. Where is the loss function implemented ?

**Comment by Jintao-Huang** [comment]
In liger_kernel.

I suggest disabling liger_kernel for debugging.
